hack/install-deps.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# shamelessly adapted from https://github.com/vmware-tanzu/carvel/blob/develop/site/static/install.sh
+
+if test -z "$BASH_VERSION"; then
+  echo "Please run this script using bash, not sh or any other shell." >&2
+  exit 1
+fi
+
+install() {
+  set -euo pipefail
+
+  dst_dir="${K14SIO_INSTALL_BIN_DIR:-/usr/local/bin}"
```

Comment:
Since this script is new, it might be nice to rebrand with something Carvel as opposed to k14s.

Reply:
I like it, but my inclination is to maintain "backwards" compatibility since it's a 'new' script ripped off of an old script.  bash seems to have no issues nesting the little `:-` default-if-null operator.  e.g.  (in my environment the K14 env var isn't set,  but the EDITOR is) : 

```
➜  carvel-kapp-controller git:(develop) echo "${K14SIO_INSTALL_BIN_DIR:-${EDITOR:-/usr/local/bin}}"                                  
nvim
➜  carvel-kapp-controller git:(develop) echo "${K14SIO_INSTALL_BIN_DIR:-${CARVEL_ALSO_NOT_SET:-/usr/local/bin}}"                     
/usr/local/bin
```